boards/wio_terminal/src/display.rs

```diff
@@ -91,8 +87,8 @@ impl Display {
         // does not appear to support PWM.
         //   HIGH - backlight enabled
         //   LOW  - backlight disabled
-        let mut backlight = self.backlight.into_push_pull_output(port);
-        backlight.set_high()?;
+        let mut backlight = self.backlight.into_push_pull_output();
+        backlight.set_high().ok();
```

Comment:
any reason not to still return the error like before? I know it's impossible, but 🤷🏼

Reply:
I think that's a question for @jbeaurivage. I didn't make that change.

Reply:
This case is less clear. There are already some errors that are passed back to the user, but they are completely erased and converted to `()`, so I don't really know what a user is supposed to do with them other than `panic!`. Seems like someone might want to rethink this approach.

In any event, I think that's a problem for another day. I vote to leave the current `.ok()` approach, because it signals that setting the pin is actually `Infallible`.